rcl_interfaces/msg/LoggerLevel.msg

```diff
@@ -0,0 +1,13 @@
+# All available logger levels
```

Comment:
We should note here that this will match what is in rcutils.  Maybe something like:

```suggestion
# All available logger levels; these correspond to the enums in rcutils
```

Reply:
Updated